apps/backend/tsconfig.json

```diff
@@ -1,7 +1,11 @@
 {
 	"extends": "../../tsconfig.json",
 	"files": ["src/index.d.ts"],
-	"include": ["**/*.ts", "**/*.js"],
+	"include": [
+		"**/*.ts",
+		"**/*.js",
+		"src/libs/constants/min-git-emails-length-for-split.constant.ts"
```

Comment:
Oops. This shouldn't be here ;)

Reply:
Good question, I even don't know why it is here